.github/actions/upload-artifact/action.yml

```diff
@@ -18,7 +18,7 @@ runs:
         tar -zcvf ${{ inputs.name }}.tar.gz ${{ inputs.path }}
       shell: bash
     - name: Upload artifact
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
Do we have any hidden files in our artifacts? so the question is, does [this](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#hidden-files) apply to us?

Reply:
great question! We do have a few hidden files like `.stylelintrc` or `.gitignore`. But those should not be relevant to the build artifacts that we're moving around here.

Reply:
Just wanted to double check. Searching through the build results I'm also seeing no hidden files either.